pages/entropy/_meta.json

```diff
@@ -47,6 +47,7 @@
     "newWindow": true
   },
   "examples": "Example Applications",
+  "request-variants": "Request Variants",
```

Comment:
The index title is not self explanatory.

Reply:
Request callback variants?

Reply:
sure, changed the title to Request callback variants.